slick/src/sphinx/gettingstarted.rst

```diff
@@ -61,9 +62,10 @@ log output.
 The Reactive Streams API is pulled in automatically as a transitive dependency.
 
 If you want to use Slick's connection pool support for HikariCP_, you need to add
-the ``slick-hikaricp`` module as a dependency in the same way as shown for ``slick``
-above. It will automatically provide a compatible version of HikariCP as a transitive
-dependency.
+the ``slick-hikaricp`` module as a dependency in the same way as shown above. It
```

Comment:
The phrase "in the same way" doesn't add anything here. I suggest cutting it so the line can read: "as a dependency, as shown above".